assets/src/dashboard/components/multiPartPill/index.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const PillContainer = styled.div`
+  display: inline-flex;
+  align-items: center;
+  height: 44px;
+  padding: 0 10px;
+  border-radius: ${({ theme }) => theme.border.buttonRadius};
+  box-shadow: 0px 2px 11px rgba(0, 0, 0, 0.25);
+`;
+
+const PillSeparator = styled.div`
+  width: 1px;
+  height: 20px;
+  margin: 0 10px;
+  background-color: ${({ theme }) => theme.colors.gray75};
+`;
+
+function MultiPartPill({ children }) {
+  const parts = useMemo(
+    () => (Array.isArray(children) ? [...children] : [children]),
+    [children]
+  );
+
+  return (
+    <PillContainer>
+      {parts.map((part, index) =>
+        index > 0 ? (
+          <React.Fragment key={index}>
+            <PillSeparator />
```

Comment:
nit: this style, i think, can be achieved with `:after` (or `:before`)  but the div works well too

Reply:
I think you're right!  I originally was trying to do something more complicated but then realized it was much easier and didn't take it all the way back to pure css.  I'll give it a shot!

Reply:
yea for these things I'm a big fan of doing something like:
```
const Styled = styled.something`
  // ... styles applied to all but first element
  :first-of-type {
    // revert styles you don't want applied to first el
  }
`
```

Reply:
I had to wrap each `part` in a `PillPart` div so I could get the `:before` rendering correctly, but I got it working!  Same divider goodness with just css 🎉  thanks for the callout!